### PR TITLE
Modify return value in example for coherence

### PR DESCRIPTION
**Pull request by 8U61ife**

### CODE REVIEW

File: examples/rvv_memcpy.c [hunk 1]
```diff
@@ -27,4 +29,4 @@ int main() {
 
   // compare
   puts(compare_1d(golden, actual, N) ? "pass" : "fail");
-}
```
Comment (eopXD) [inline examples/rvv_memcpy.c:30]: May I ask why is this line updated?

**Review by eopXD — APPROVED** [review]
LGTM except the minor comment.

File: examples/rvv_memcpy.c (hunk 1 above)
Comment (8U61ife) [reply examples/rvv_memcpy.c:30]: In fact, I don't know.I just use vim to edit.

Comment (8U61ife) [reply examples/rvv_memcpy.c:30]: It seems that there is a newline character which auto apeend in my file. Although I try to press `delete` at the end.

Comment (eopXD) [reply examples/rvv_memcpy.c:30]: Please remove it as this is not related to the topic of this PR. Coding style issues shall be dealt in a separately.

Comment (8U61ife) [reply examples/rvv_memcpy.c]: Done.

**Comment by eopXD** [comment]
Thanks for the PR.

**Pull request merged**